src/client/dfuse/dfuse.h

```diff
@@ -563,6 +569,9 @@ struct dfuse_inode_entry {
 	d_list_t                 ie_htl;
 
 	/* Time of last kernel cache update.
+	 * For directories this is the time of the most recent closedir for a handle which may
+	 * have populated the cache.
+	 * For files this is when the file was last closed after been written to.
```

Comment:
```suggestion
	 * For files this is when the file was last closed after being written to.
```

Reply:
I've pulled these two grammar corrections into another PR, will update in due course.